modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaExtractTextCheck.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.CharPool;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Nícolas Moura
+ */
+public class UpgradeJavaExtractTextCheck extends BaseFileCheck {
```

Comment:
Because not all methods of HTMLutil goes to HtmlParser this is the only method that should be replaced,  because of that the name is exactly the method. Htmlutil still exists and has the methods, this was the one that was replaced

Reply:
Even still, `UpgradeJavaHtmlUtilCheck` would be a better name. `UpgradeJavaExtractTextCheck` is still, in my opinion, too ambiguous. If you want, you could use the name `UpgradeJavaHtmlUtilExtractTextCheck`.